applications/GeoMechanicsApplication/tests/test_dynamics_long.py

```diff
@@ -0,0 +1,107 @@
+import os
+import json
+
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+
+import test_helper
+from strip_load_semi_analytical_solution import StripLoad
+
+
+class KratosGeoMechanicsDynamicsLongTests(KratosUnittest.TestCase):
+    """
+    This class contains long-running tests to check dynamic calculations
+    """
+
+    def test_constant_strip_load_2d(self):
+        """
+        Tests a constant strip load on a 10m X 10m  block. The block is loaded with a constant strip load of 1kN/m and
+        a length of 1m.
+
+        The solution should roughly follow the semi-analytical solution presented in the following publication:
+        An Introduction to Soil Dynamics , Verruijt A., 2009, Delft University of Technology, Chapter 12.2
+
+
+        """
+        # set to true if the results should be checked with the analytical solution
+        CHECK_RESULTS = False
```

Comment:
If its true, the analytical solution will be calculated and there will be a plot using matplotlib. Since matplotlib is not a kratos dependecy, I dont want it to be ran on the CI. 

When "CHECK_RESULTS" is false, there will still be a regression test, but no plot